apps/src/utils.js

```diff
@@ -289,6 +289,7 @@ export function getPegasusHost() {
       var name = window.location.hostname.split('.')[0];
       switch (name) {
         case 'localhost':
+        case 'localhost-studio':
```

Comment:
Aside: Shouldn't `localhost-studio` return the studio path and not the non-studio path?